float64[5] IRdata
int16[5] IRangles
